core_integrations/yale_home

```diff
@@ -1 +1 @@
-../core_brands/yale
```

Comment:
Shouldn't this be symlinked to the brands one as well?

Reply:
Yes I think thats better

Reply:
ac9d5b4b8f75fe756a3bd13a471040d55d9ec776